pkg/apis/core/validation/events.go

```diff
@@ -117,13 +116,5 @@ func isNamespacedKind(kind, groupVersion string) (bool, error) {
 		return false, err
 	}
 
-	restMapping, err := g.RESTMapper.RESTMapping(schema.GroupKind{Group: gv.Group, Kind: kind}, gv.Version)
```

Comment:
would this have errored if you asked about a kind that is not registered into this group? do we need to do a similar check before answering `false, nil` about a kind not even known to the group?

Reply:
ok